deposit/src/test/java/edu/unc/lib/deposit/validate/VirusScanJobTest.java

```diff
@@ -233,6 +234,49 @@ public void errorScanTest() throws Exception {
         }
     }
 
+    @Test
+    public void pauseScanTest() throws Exception {
+        when(scanResult.getStatus()).thenReturn(Status.PASSED);
+
+        Model model = job.getWritableModel();
+        Bag depBag = model.createBag(depositPid.getRepositoryPath());
+
+        File pdfFile = new File(depositDir, "pdf.pdf");
+        File textFile = new File(depositDir, "text.txt");
+        PID file1Pid = addFileObject(depBag, pdfFile);
+        PID file2Pid = addFileObject(depBag, textFile);
+
+        // Should be running for the first file, then paused
+        when(depositStatusFactory.getState(depositUUID))
+                .thenReturn(DepositState.running)
+                .thenReturn(DepositState.running)
```

Comment:
Same here, does it need to be here twice?

Reply:
this was so it would register as "paused" on the second file, since there is now an interrupt check at the beginning of every deposit job.